Readme.md

```diff
@@ -143,16 +143,34 @@ $ LIBRARY_PATH="[gcc-path value]" LD_LIBRARY_PATH="[gcc-path value]" rustc +$(ca
 ## Env vars
 
 <dl>
-    <dt>CG_GCCJIT_INCR_CACHE_DISABLED</dt>
-    <dd>Don't cache object files in the incremental cache. Useful during development of cg_gccjit
-    to make it possible to use incremental mode for all analyses performed by rustc without caching
-    object files when their content should have been changed by a change to cg_gccjit.</dd>
+    <dt>CG_GCCJIT_DUMP_ALL_MODULES</dt>
+    <dd>Enables dumping of all compilation modules. When set to "1", a dump is created for each module during compilation and stored in `/tmp/reproducers/`.</dd>
+    <dt>CG_GCCJIT_DUMP_MODULE</dt>
+    <dd>Enables dumping of a specific module. When set with the module name, e.g., `CG_GCCJIT_DUMP_MODULE=module_name`, a dump of that specific module is created in `/tmp/reproducers/`.</dd>
     <dt>CG_GCCJIT_DISPLAY_CG_TIME</dt>
```

Comment:
This seems unused, so please remove.

Reply:
The variables you removed are used.
You should add them back and remove `CG_GCCJIT_DISPLAY_CG_TIME` instead.